Universal-Federated-Analytics.js

```diff
@@ -271,7 +271,7 @@ Digital Analytics Program Government Wide Site Usage Measurement and Tracking.
        */
       (function () {
         var WVscript = document.createElement('script');
```

Comment:
This of course would be `https://dap.digitalgov.gov/web-vitals/dist/web-vitals.attribution.iife.js` in the real code.

Reply:
@sanason This was tested, and ready good to go. #153 is ready to go. it has the production link of webvitals you provided, minified, and source mapped.